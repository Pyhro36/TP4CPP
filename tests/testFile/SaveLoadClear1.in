PC convex0 5 6 3 6 3 -4 5 -4
PC convex1 1 0 1 1 0 1 0 0
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1
S seg0 0 4 10 0
S seg1 0 0 10 4
R rec0 0 4 10 0
PC pc 0 4 10 0 0 0
OI oi seg0 seg1 rec0 pc
S seg2 0 4 10 0
R rec -10 -5 1 5
OR or seg2 rec
LIST
DELETE rec rec0
LIST
SAVE testFile/testFILE.txt
CLEAR
LIST
LOAD testFile/testFILE.txt
LIST
CLEAR
LIST
EXIT