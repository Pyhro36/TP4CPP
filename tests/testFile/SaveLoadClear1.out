Shape handler 1.0 :
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
PC convex0 5 6 3 6 3 -4 5 -4
PC convex1 1 0 1 1 0 1 0 0
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1
S seg0-oi 0 4 10 0
S seg1-oi 0 0 10 4
R rec0-oi 0 4 10 0
PC pc-oi 0 4 10 0 0 0
OI oi seg0-oi seg1-oi rec0-oi pc-oi
DELETE seg0-oi seg1-oi rec0-oi pc-oi
S seg2-or 0 4 10 0
R rec-or -10 -5 1 5
OR or seg2-or rec-or
DELETE seg2-or rec-or
PC pc 0 4 10 0 0 0
R rec -10 -5 1 5
R rec0 0 4 10 0
S seg0 0 4 10 0
S seg1 0 0 10 4
S seg2 0 4 10 0

OK
PC convex0 5 6 3 6 3 -4 5 -4
PC convex1 1 0 1 1 0 1 0 0
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1
S seg0-oi 0 4 10 0
S seg1-oi 0 0 10 4
R rec0-oi 0 4 10 0
PC pc-oi 0 4 10 0 0 0
OI oi seg0-oi seg1-oi rec0-oi pc-oi
DELETE seg0-oi seg1-oi rec0-oi pc-oi
S seg2-or 0 4 10 0
R rec-or -10 -5 1 5
OR or seg2-or rec-or
DELETE seg2-or rec-or
PC pc 0 4 10 0 0 0
S seg0 0 4 10 0
S seg1 0 0 10 4
S seg2 0 4 10 0

OK
OK

OK
PC convex0 5 6 3 6 3 -4 5 -4
PC convex1 1 0 1 1 0 1 0 0
PC convex2 17 4 8 0 2 0 0 7 2 10 11 11
PC convex3 -8 -6 -2 -10 1 -4 -5 -1
S seg0-oi-oi 0 4 10 0
S seg1-oi-oi 0 0 10 4
R rec0-oi-oi 0 4 10 0
PC pc-oi-oi 0 4 10 0 0 0
OI oi seg0-oi-oi seg1-oi-oi rec0-oi-oi pc-oi-oi
DELETE seg0-oi-oi seg1-oi-oi rec0-oi-oi pc-oi-oi
S seg2-or-or 0 4 10 0
R rec-or-or -10 -5 1 5
OR or seg2-or-or rec-or-or
DELETE seg2-or-or rec-or-or
PC pc 0 4 10 0 0 0
S seg0 0 4 10 0
S seg1 0 0 10 4
S seg2 0 4 10 0

OK

